code/modules/antagonists/changeling/powers/fakedeath.dm

```diff
@@ -19,7 +19,8 @@
 	..()
 	if(revive_ready)
 		INVOKE_ASYNC(src, PROC_REF(revive), user)
-		disable_revive(user) // this should be already called via signal, but just incase something wacky happens
+		//removed the double call to disable_revive, it's already called in revive() via cure_fakedeath
+		//calling to the REMOVE_TRAIT signal
```

Comment:
historical comment is not useful here, kindly remove it. anyone with a stake in the code can see revive() do it by looking at the invoke_async at the line above (and this comment will likely degrade quickly should the actual code change and no one fix the comment)